src/coreclr/src/inc/CrstTypes.def

```diff
@@ -351,6 +351,9 @@ End
 Crst JitGenericHandleCache
 End
 
+Crst JitPatchpoint
```

Comment:
Maybe could reuse `CrstLeafLock` instead, as there appear to be are many different locks in that class

Reply:
Can no longer be a leaf lock as we have to allocate while holding.